apps/mdn/mdn-aws/infra/mdn-cdn/mdn_cdn.tf

```diff
@@ -32,3 +33,15 @@ module "mdn-cloudfront-prod" {
 }
 
 
+# Attachments origin
+
+module "mdn-cloudfront-attachments-prod" {
+    source = "./cloudfront_attachments"
+
+    acm_cert_arn = "arn:aws:acm:us-east-1:236517346949:certificate/4322464c-b6d9-44d9-914d-3554461bcbb3"
```

Comment:
``us-east-1``? Is this right?

Reply:
all cloudfront certs have to be provisioned in us-east-1.